manifests/init.pp

```diff
@@ -100,7 +100,7 @@
 # @param service_name
 #   **Private** You should not need to override this parameter.
 class nrpe (
-  Array[Stdlib::Host]                  $allowed_hosts                   = ['127.0.0.1'],
+  Array[Variant[Stdlib::Fqdn,Stdlib::IP::Address]]                  $allowed_hosts                   = ['127.0.0.1'],
```

Comment:
```suggestion
  Array[Variant[Stdlib::Fqdn,Stdlib::IP::Address]]  $allowed_hosts                   = ['127.0.0.1'],
```

can you please clean it up a bit like this this and maybe remove a bit of the whitespace so the `=` are vertically aligned?

Reply:
This was unintended, this is now fixed.